react/README.md

```diff
@@ -531,7 +531,7 @@
 
     // good
     render() {
-      return (<div />);
```

Comment:
Please do; our rules only don't require parens for single-line jsx because there isn't internal consensus at Airbnb on requiring them nor forbidding them - thus, it's important that this guide *be* inconsistent, to illustrate that either is perfectly acceptable.